Locationapp/app/src/main/java/com/example/manya/locationapp/MainActivity.java

```diff
@@ -0,0 +1,20 @@
+package com.example.manya.locationapp;
+
+import android.support.v7.app.AppCompatActivity;
+
+import android.os.Bundle;
+import android.support.v7.widget.LinearLayoutManager;
+import android.support.v7.widget.RecyclerView;
+public class MainActivity extends AppCompatActivity {
+    private PlacesListAdapter Adapter;
```

Comment:
Readup [this](https://github.com/ribot/android-guidelines/blob/master/project_and_code_guidelines.md) for naming and code guidelines.